src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -19,10 +19,56 @@ This section describes major APIs that were in common use. For information about
 
 ### Preferences Framework ###
 
+#### Model Changes ####
+
+##### Reset ######
+By default, reset will only reset the `preferences` model path. Other model values will remain unchanged. If you'd like to also have these paths changed, add a listener to the `beforeReset` event to execute a fireChangeRequest for the model paths you need to reset. (See: [ArrowScrolling.js](https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/ArrowScrolling.js))
+
+```javascript
+\\...
```

Comment:
The `\\` at the start of the line shows as is on the web page. Probably remove this line. Same to line 35.